.github/workflows/python-package.yml

```diff
@@ -133,7 +133,7 @@ jobs:
     strategy:
         fail-fast: false
         matrix:
-            python-version: ['3.8', '3.9', '3.10', '3.11', '3.12']
+            python-version: ['3.8', '3.12']
```

Comment:
smart move

Reply:
Thanks bro!
I’m dying inside from all of these CI failures haha.. We need a clean CI and I won’t let it go until we have it 💪